tests/app/celery/test_ftp_update_tasks.py

```diff
@@ -91,6 +95,20 @@ def test_update_letter_notifications_statuses_builds_updates_list(notify_api, mo
     assert updates[1].cost_threshold == 'Sorted'
 
 
+def test_update_letter_notifications_statuses_persisted(notify_api, mocker, sample_letter_template):
```

Comment:
Add tests for neither Sent or Failed?

Reply:
At the moment if it's not `Sent`, then it's assumed to be Failed and set to `failed` status. 
If we don't get a `Sent` back from DVLA then I'm not sure if there is a better status to set it to other than `failed`